samba-manager/files/samba-manager.init

```diff
@@ -98,8 +102,11 @@ parse_config() {
 	config_get sharename ${cfg} sharename
 
 	if [ $enabled -eq 1 ] ; then
+		if [ -z $password ] ; then
```

Comment:
I think we need to quote the $password here?
`[ -z "$password" ]`

Reply:
> I think we need to quote the $password here? `[ -z "$password" ]`

Changed.